packages/cli/src/lib/pg/setter.ts

```diff
@@ -0,0 +1,62 @@
+import {APIClient} from '@heroku-cli/command'
+import {ux} from '@oclif/core'
+import {addonResolver} from '../addons/resolve'
+import host from './host'
+import {essentialPlan} from './util'
+
+type BooleanAsString = true | false | 'on' | 'ON' | 'true' | 'TRUE' | 'off' | 'OFF' | 'false' | 'FALSE'
+
+export const boolean = (value: BooleanAsString) => {
+  switch (value) {
+  case 'true':
+  case 'TRUE':
+  case 'ON':
+  case 'on':
+  case true:
+    return true
+  case 'false':
+  case 'FALSE':
+  case 'OFF':
+  case 'off':
+  case null:
+  case false:
+    return false
+  default:
+    throw new TypeError('Invalid value. Valid options are: a boolean value')
+  }
+}
+
+export const numeric = (value: string | number) => {
```

Comment:
Like `boolean` I think commands that require this could just be replaced with:
```javascript
  value: Args.integer()
```

Reply:
+1